keras_cv/models/object_detection/yolo_v8/yolo_v8_detector.py

```diff
@@ -474,12 +473,14 @@ def compile(
             raise ValueError("User metrics not yet supported for YOLOV8")
 
         if isinstance(box_loss, str):
-            if box_loss == "iou":
-                box_loss = YOLOV8IoULoss(reduction="sum")
+            if box_loss == "ciou":
+                box_loss = CIoULoss(bounding_box_format="xyxy", reduction="sum")
+            elif box_loss == "iou":
+                box_loss = IoULoss(bounding_box_format="xyxy", reduction="sum")
```

Comment:
To make backwards compat happy, really this should be CIoU loss, although maybe it's best to just allow this as a breaking change since it's a lot more intuitive as an API this way.

So I think I'm happy with this. However, let's print a warning in the "iou" case that says 

```
warnings.warn("YOLOV8 recommends using CIoU loss, but was configured to use standard IoU. Consider using `box_loss='ciou'` instead.")
```